File: ClientApp/app/core/domain/message.ts
export class  Message {
    Id:number;
    ChatId:number;
    Date: string;
    SenderId:number;
    Text:string;
    SenderFirstName: string;
    SenderLastName:string;
    Photo:string;
    constructor(id:number, chatId:number, date:string, senderId:number, text:string, senderFirstName:string,
      senderLastName:string, senderPhoto:string) {
        this.Id=id;
        this.ChatId=chatId;
        this.Date=date;
        this.SenderId=senderId;
        this.Text=text,
        this.SenderFirstName=senderFirstName
        this.SenderLastName=senderLastName;
        this.Photo=senderPhoto;
    }
}

File: Models/Message.cs
namespace signalr_test.Models
{
    public class Message : IEntityBase
    {
        public int Id { get; set; }
        public string Text { get; set; }
        public string Date { get; set; }
        public int SenderId { get; set; }
        public int ChatId { get; set; }

    }
}
